### PR TITLE
gh-actions: fix hard-coded PDF name in deploy action

### LINKED ISSUE #1125: build: deploy workflow has a hard-coded file-name reference to the Guidelines PDF
https://github.com/music-encoding/music-encoding/blob/382f64072f53dc6062283383c1d80c7d9b00dbf1/.github/workflows/deploy.yml#LL107C50-L107C50

### PR DESCRIPTION
**Pull request by bwbohl**
* updates actions/upload-artifacs to v3.1.2 which introduces globs in artifact path specification
* whitespace linting

### CODE REVIEW

**Review by musicEnfanthen — DISMISSED** [review]

**Comment by musicEnfanthen** [comment]
Thank you!

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -42,19 +42,19 @@ jobs:
...
-      
+
       - name: Build schema and guidelines with Docker image
         run: docker run --rm -v $(pwd):/opt/docker-mei/music-encoding ghcr.io/music-encoding/docker-mei:latest ant -noinput -buildfile music-encoding/build.xml -Ddocker=true -Dgithub.sha=${{ github.sha }} -Dgithub.event_name=${{ github.event_name }} -Dgithub.ref=${{ github.ref }}
 
-      ### UPLOADING THE ARTIFACTS ### 
+      ### UPLOADING THE ARTIFACTS ###
       - name: Upload PDF
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v3.1.2
```
Comment (rettinghaus) [inline .github/workflows/deploy.yml]: Why change this? `@v3` will take the latest version 3 anyway.

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -42,19 +42,19 @@ jobs:
...
         with:
           name: pdf
-          path: ${{ github.workspace }}/dist/guidelines/pdf/MEI_Guidelines_v5.0.0-dev.pdf
-      
+          path: ${{ github.workspace }}/dist/guidelines/pdf/MEI_Guidelines_*.pdf
+
       - name: Upload Prince XML log
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v3.1.2
```
Comment (rettinghaus) [inline .github/workflows/deploy.yml]: Also here.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (bwbohl) [reply .github/workflows/deploy.yml]:
```suggestion
        uses: actions/upload-artifact@v3
```

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (bwbohl) [reply .github/workflows/deploy.yml]:
```suggestion
        uses: actions/upload-artifact@v3
```

**Review by rettinghaus — APPROVED** [review]

**Review by musicEnfanthen — APPROVED** [review]

**Pull request merged**